### DISCUSSION TITLE
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found

### DISCUSSION THREAD

**Discussion by gameveloster** [ggml-org/llama.cpp#9688] [Q&A]
Downloaded and unzipped the [latest release `llama-b3841-bin-ubuntu-x64.zip`](https://github.com/ggerganov/llama.cpp/releases/tag/b3841) on my Ubuntu 22.04 system.

When I tried to run `llama-gguf-split`, I get a `GLIBC` error:

```bash
$ ./llama-gguf-split
./llama-gguf-split: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by ./llama-gguf-split)
```

Was my installation not performed correctly? Looking for help, thank you

**Comment by HuangShumin** [comment]
libc.so.6: version `GLIBC_2.38' is available on Ubuntu 24.04

**Comment by ztsweet** [comment]
+1
